UserSettings.xml

```diff
@@ -0,0 +1 @@
+﻿<?xml version="1.0" encoding="utf-8"?><Settings xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><!-- This file is for local user settings and preferences. --><SqlAzureSettingsUserFileCreated>20220301055501</SqlAzureSettingsUserFileCreated></Settings>
```

Comment:
Should this file be checked in?

Reply:
I think no though I'm not quite sure what generated this file, I'll delete it.
As it says `<!-- This file is for local user settings and preferences. -->` maybe we can add it to .gitignore